lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -137,9 +137,16 @@ class CatalogController < ApplicationController
     ##
     # Configuration for partials
     config.index.partials.insert(0, :index_breadcrumb)
-    
-    config.show.metadata_partials = [:summary_field, :access_field]
-    
+
+    config.show.metadata_partials = [
+      :summary_field,
+      :access_field,
+      :background_field,
+      :scope_and_arrangement_field,
+      :related_field,
+      :admin_info_field
+    ]
```

Comment:
Ah.. yes I believe they will. At some point we will need to change:
```diff
- config.show.metadata_partials = [...]
+ config.show.collection.metadata_partials = [...]
+ config.show.component.metadata_partials = [...]
```
Or something like that